package.json

```diff
@@ -23,18 +22,18 @@
     "prepublishOnly:pull": "git pull",
     "prepublishOnly:install": "npm ci",
     "prepublishOnly:test": "npm test",
-    "test": "jest"
+    "test": ""
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/netlify/netlify-plugin-angular-universal.git"
+    "url": "git+https://github.com/netlify/angular-runtime.git"
   },
   "author": "@lindsaylevine",
   "license": "ISC",
```

Comment:
Should be MIT licence. You can rmeove the author field too now

Reply:
done in https://github.com/netlify/netlify-plugin-angular-universal/pull/67/commits/8e84883603d27c65b47d21b426dbf3193e4ea5b1